### PR TITLE
[service.ifttt] 1.0.0

### PR DESCRIPTION
**Pull request by plucka3**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: service.ifttt/service.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+import json, urllib2
+
+__addon__ = xbmcaddon.Addon()
+__addonversion__ = __addon__.getAddonInfo('version')
+__addonid__ = __addon__.getAddonInfo('id')
+__addonname__ = __addon__.getAddonInfo('name')
+__addonPath__ = __addon__.getAddonInfo('path')
+__addonResourcePath__ = xbmc.translatePath(os.path.join(__addonPath__, 'resources', 'lib'))
+__addonIconFile__ = xbmc.translatePath(os.path.join(__addonPath__, 'icon.png'))
+sys.path.append(__addonResourcePath__)
```
Comment (enen92) [inline service.ifttt/service.py:12]: Please use relative imports if possible instead of the append to the sys.path

Comment (plucka3) [reply service.ifttt/service.py:12]: Im not really sure how to change this can you provide an example.

Comment (enen92) [reply service.ifttt/service.py:12]: Just do `from resources.lib import whateveryouwant`

You might need a placeholder/empty  `__init__.py` file in resources, and another one in lib

Comment (plucka3) [reply service.ifttt/service.py:12]: Still no idea what youre asking, this is a standard bit of code from the base templates I didnt write it and dont understand what youre saying, please be specific or point me to a sample project that does this as your expecting.

Comment (enen92) [reply service.ifttt/service.py:12]: See https://github.com/xbmc/repo-plugins/pull/3074#pullrequestreview-472262294 for a similar example

**Pull request closed without merging**